Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -125,6 +125,80 @@ final class CacheAdvanceTests: XCTestCase {
         }
     }
 
+    private func prepareCacheWithIncorrectOffsetInHeader() throws -> CacheHeaderHandle {
+        let randomHighValue: UInt64 = 10_1000
```

Comment:
Does this high value need to be "high"? Or does it just need to be higher than the length of `TestableMessage.lorumIpsum[0]`? We have a calculation for `requiredByteCount` method in this file – could we use that?

Reply:
I copied the code from [test](https://github.com/dfed/CacheAdvance/pull/64/files#diff-9de095b2bcc9ceaefec7d7de77b3a8f1d5b05e4ae82ba0da807b213ea076ebadL100) above: "func test_messages_canReadEmptyCacheThatDoesNotOverwriteOldestMessages" 

@dfed

Reply:
🤦‍♂️ sorry I was reviewing too quickly last night. Thank you for calling out what you were doing in a comment here.

I remember writing that test now. Let me re-review this part of the code with that context in mind 🙂